Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.cxx

```diff
@@ -2061,11 +2089,22 @@ ::CreateTransformedClipper(vtkMRMLTransformNode *tnode, vtkMRMLModelNode::MeshTy
     {
     if (type == vtkMRMLModelNode::UnstructuredGridMeshType)
```

Comment:
Th mesh type check is because we are currently using vtkClipPolyData for surface meshes and vtkClipDataSet for volumetric.  My code currently doesn't implement whole cell clipping for sruface meshes (will fix that!), but that would similarly use vtkExtractPolyDataGeometry or vtkExtractGeometry.  ClipDataSet and ExtractGeometry can be used on PolyData, but they always return an unstructuredGrid data type.  I don't know if that would be a problem or not.